src/_includes/software_modernisation_curated_publications.html

```diff
@@ -1,3 +1,3 @@
 {% assign translations = site.translations[site.lang].curated-publications.softmod %}
 
-{% include curated_publications.html tag="software modernisation" title=translations.intro-title description=translations.intro-description %}
+{% include curated_publications.html tag="software modernisation" title=translations.intro-title description=translations.intro-description class='software-modernisation' %}
```

Comment:
```suggestion
{% include curated_publications.html tag="software modernisation" title=translations.intro-title description=translations.intro-description class='curated-publications--software-modernisation' %}
```

Reply:
Reckon this is resolved via: https://github.com/codurance/site/pull/1730/commits/c7fb72f5cc9c11aa808e292b229b1b92de03927c